buildspec/linuxUiTests.yml

```diff
@@ -25,6 +25,9 @@ phases:
       dotnet: 3.1
 
     commands:
+      # WORKAROUND for yarn key being out of date, update keys so we can run builds TODO remove dirmngr and apt-key commands
+      - apt-get install dirmngr
+      - apt-key adv --refresh-keys --keyserver keyserver.ubuntu.com
```

Comment:
Is this preferred over `curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -` for any reason??

Reply:
I mean, I prefer not to curl a random key and add it directly. Other than that, no